truss/tests/test_model_inference.py

```diff
@@ -760,6 +760,92 @@ def _test_invocations(expected_code):
         _test_invocations(200)
 
 
+@pytest.mark.integration
+def test_setup_environment():
+    """
+    Tests a Truss that has a setup_environment function defined, and ensures that it
+    gets called whenever changes are made to the dynamic_config environment key.
+    To do this, we run an inference that modifies the file and then sleep for ten seconds
+    to allow for the file watcher to notice the updates to the mounted configmap.
+    """
+    model = """
+    class Model:
+        def setup_environment(self, environment: dict):
+            print("setup_environment called with", environment)
+            environment_name = environment.get("environment_name", None)
+            if environment_name == "production":
+                print("DOING IT LIVE")
```

Comment:
probably ned to clean this up

Reply:
maybe just "in production environment" and "in beta environment"